ddtrace/internal/module.py

```diff
@@ -241,6 +242,8 @@ def get_code(_loader, fullname):
 
                 return code
 
+            get_code.__setattr__("_dd_get_code", True)
```

Comment:
```suggestion
            setattr(get_code, "_dd_get_code", True)
```

is there a difference? we use `getattr` above 🤷🏻